### PR TITLE
Use palette colors for default bg/fg

### LINKED ISSUE #373: Dark style is low constrast
The default color scheme for the terminal is low contrast. I'm not sure if it's the somewhat small font size or the dull background color, but some combo of factors is making things a bit more difficult to read in the terminal than other apps. It is most evident in sunlight.

Here I put elementary and gnome terminal apps side-by-side to make the difference in contrast clear. 

Screenshot from 2019-05-14 19-32-38

### PR DESCRIPTION
**Pull request by cassidyjames**
Better matches https://github.com/elementary/stylesheet/pull/510, plus better contrast.

## Before

image

image

## After

image

image
 
Fixes #373

### CODE REVIEW

**Comment by fabiozaramella** [comment]
This makes the text much more readable.
Has a decision been reached about this?

**Comment by cassidyjames** [comment]
Now that the neutral GTK style has been shipping for a while, it'd be great to get the Terminal to match. Right now it's off and it feels weird imho.

File: data/io.elementary.terminal.gschema.xml [hunk 1]
```diff
@@ -91,7 +91,7 @@
     </key>
 
     <key name="foreground" type="s">
-      <default>"#94a3a5"</default>
```
Comment (jlnr) [inline data/io.elementary.terminal.gschema.xml]: This color is still being referenced in `io.elementary.terminal.gschema.xml`, as part of "palette", and in the method `TerminalWidget.restore_settings`. Should these be replaced as well, or should the palette be left untouched by this PR? (Not sure why there is both a palette, and default fg/bg colors.)

File: data/io.elementary.terminal.gschema.xml [hunk 2]
```diff
@@ -103,7 +103,7 @@
       </description>
     </key>
     <key name="background" type="s">
-      <default>"rgba(37, 46, 50, 0.95)"</default>
```
Comment (jlnr) [inline data/io.elementary.terminal.gschema.xml]: The `.color-dark radio` in Application.css still uses this color (with 100% opacity), should probably be changed to the gray color as well?

Comment (cassidyjames) [reply data/io.elementary.terminal.gschema.xml]: @jlnr good catch, addressed.

File: data/io.elementary.terminal.gschema.xml (hunk 1 above)
Comment (cassidyjames) [reply data/io.elementary.terminal.gschema.xml]: @jlnr hm, I'm not sure. I believe the palette is used by the VTE itself for things like curses. I'm not sure if it should be updated.

Comment (jlnr) [reply data/io.elementary.terminal.gschema.xml]: Fair enough. If it looks odd in real-world use cases, we can open another PR to fix things.

**Comment by cassidyjames** [comment]
> LGTM, does this need more UX discussion?

Since we've merged the stylesheet changes and this makes Terminal match, I think we're good. :smile:

**Pull request merged**